Tasks/AzureIoTEdgeV2/genconfig.ts

```diff
@@ -0,0 +1,41 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as fs from "fs";
+import * as path from "path";
+import util from "./util";
+import Constants from "./constant";
+import { IExecOptions } from 'azure-pipelines-task-lib/toolrunner';
+
+export async function run() {
+    let templateFilePath: string = tl.getPathInput("templateFilePath", true);
+    tl.debug(`The template file path is ${templateFilePath}`);
+    if (!fs.existsSync(templateFilePath)) {
```

Comment:
can we use async version?

Reply:
The async version is marked as deprecated

Reply:
https://iojs.org/api/fs.html#fs_fs_access_path_mode_callback seems now recommend to use access?

Reply:
as discussed offline, keep using the existsSync method